### ISSUE TITLE
The "Passing arguments by reference correctness" inspection doesn't catch object creation

### ISSUE THREAD

**Issue by aik099** [kalessil/phpinspectionsea#1081]
First, thank you for reporting a bug or making a request. That takes time and we appreciate that!

Let's start with common details:
 
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Issue type     | Bug                          |
| Plugin         | Php Inspections (EA Extended), 3.0.7 |
| Language level | PHP 5.4                        |

Current behaviour (description/screenshot:)
--

The following code isn't reported as error by `PHP | Php Inspections (EA Extended) | Probable bugs | Passing arguments by reference correctness` inspection:

```php
$this->Application->HandleEvent( new kEvent($prefix . ':OnAfterConfigRead') );
```

Expected behaviour
--

The above code should be reported with this error:

```
Emits a notice (only variable references should be returned by reference).
```
Example code, that is reported correctly is: `end(explode('/', $imageLink));`.

Environment details
---
Information from Help -> About dialog (contains IDE and it's version details).

PhpStorm 2018.2.5
Build #PS-182.4892.16, built on October 11, 2018
Licensed to Alexander Obuhovich
Subscription is active until June 29, 2019
JRE: 1.8.0_152-release-1248-b8 x86_64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
macOS 10.14

Notes
---
- Bugs fixed with the highest priority, so features might be postponed. But it shouldn't stop you, right?
- The project support is powered by community funding via [Patreon](https://www.patreon.com/kalessil)

**Comment by kalessil** [comment]
Will check, thank you for reporting @aik099.

**Comment by kalessil** [comment]
Fixed!

**Comment by aik099** [comment]
Thanks for fixing. What is the release strategy for this project, like every month or sooner?
